src/main/java/com/nowoczesnyjunior/financialapp/expensemodule/mapper/ExpenseMapper.java

```diff
@@ -0,0 +1,34 @@
+package com.nowoczesnyjunior.financialapp.expensemodule.mapper;
+
+import com.nowoczesnyjunior.financialapp.expensemodule.model.Expense;
+import com.nowoczesnyjunior.financialapp.expensemodule.model.ExpenseCategory;
+import com.nowoczesnyjunior.financialapp.openapi.model.CategoryDto;
+import com.nowoczesnyjunior.financialapp.openapi.model.ExpenseDto;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+
+import java.util.List;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING, uses = {CategoryDto.class, ExpenseCategory.class})
+public interface ExpenseMapper {
+
+    @Mapping(target = "amount", source = "amount")
+    @Mapping(target = "date", source = "expenseDate")
+    @Mapping(target = "category",source = "category" )
+    @Mapping(target = "description", source = "description")
+    ExpenseDto expenseToExpenseDto(Expense expense);
+
+    @InheritInverseConfiguration
+    Expense dtoToModel(ExpenseDto expenseDto);
+
+    List<ExpenseDto> expensesToDtos(List<Expense> expenseList);
+
+    @Mapping(target = "id", source = "categoryId")
+    @Mapping(target = "name", source = "categoryName")
+    CategoryDto categoryToDto(ExpenseCategory expenseCategory);
```

Comment:
Here the same as above, consider changing categoryId and categoryName to id and name, they are already in ExpenseCategory class after all

Reply:
done